src/ViewModels/ArcGisMapServerItemViewModel.js

```diff
@@ -43,7 +43,7 @@ defineProperties(ArcGisMapServerItemViewModel.prototype, {
      */
     type : {
         get : function() {
-            return 'esri-mapServer';
+            return 'esri-rest';
```

Comment:
Perhaps I'm being pedantic, but I think it's important to use an accurate name because it's pretty likely that NM will support very different Esri REST services in the future.  For example, their Feature Service returns vector features, so it's a bit like WFS, as opposed to a Map Service which is similar to WMS.  We'd handle that with a different view-model type, even though that one could legitimately be called `esri-rest` as well.

I had called it `esri-mapServer`, because the URL pretty much always contains "MapServer".  But it seems like `esri-mapService` would be more correct.